clang/lib/Frontend/CompilerInvocation.cpp

```diff
@@ -3306,6 +3309,19 @@ void CompilerInvocation::GenerateLangArgs(const LangOptions &Opts,
 #include "clang/Driver/Options.inc"
 #undef LANG_OPTION_WITH_MARSHALLING
 
+  if (!Opts.FPAccuracyVal.empty()) {
+    StringRef S = Opts.FPAccuracyVal;
+    GenerateArg(Args, OPT_ffp_builtin_accuracy_EQ, S, SA);
+  }
+
+  for (const auto &F : Opts.FPAccuracyFuncMap) {
+    SmallString<128> S;
+    S += F.second;
+    S += ':';
+    S += F.first;
+    GenerateArg(Args, OPT_ffp_builtin_accuracy_EQ, S, SA);
+  }
```

Comment:
```suggestion
  for (const auto &F : Opts.FPAccuracyFuncMap)
    GenerateArg(Args, OPT_ffp_builtin_accuracy_EQ, (F.second + ":" + F.first).str(), SA);
```
Can use a Twine for this, I think.